release-doc-in-process/all_vf.TTL

```diff
@@ -35,12 +35,21 @@ vf:ResourceClassification  a    owl:Class ;
 
 vf:ProcessClassification  a           owl:Class ;
         rdfs:label           "vf:ProcessClassification" ;
-        rdfs:comment   "A process category, model, taxonomy item, tag, facet value, or other kind of classification." .
+        rdfs:comment   "A process category, taxonomy item, tag, facet value, or other kind of classification." .
+
+vf:RecipeProcessClassification  a owl:Class ;
```

Comment:
I understood we agreed on just `vf:RecipeProcess` or `vf:ProcessRecipe` the second sounds better and follow the same *suffix* approach as `vf:RecipeClassification`

Reply:
@bhaugen didn't like RecipeProcess, thought it needed the Classification for clarity.  

Re. RecipeProcess vs ProcessRecipe - It is a process that is part of a recipe, the recipe includes the equivalent of processes, events, resources.  So it definitely isn't a ProcessRecipe, there is no kind of recipe included in the model explicitly, therefore no ProcessRecipe.

I'm open to either RecipeProcess or RecipeProcessClassification.  Or something that is new and better.

Reply:
Let's go for now with RecipeProcess, still for me just like we have classification of/for a process we have recipe of/for a process, we don't have a process in neither case.

Reply:
> So it definitely isn't a ProcessRecipe, there is no kind of recipe included in the model explicitly, therefore no ProcessRecipe.

So far I still like *ProcessTemplate* since we have a template for a process and crate processes from that template.

Reply:
I understood you wanted the template objects to have info that would preclude them from being used as a generic recipe for everyone.  So that seems like it could be useful, but doesn't seem like a recipe.  Let's keep thinking.....

Reply:
Not to get stuck on naming, I think we can start with *vf:RecipeProcess* and then rename it once we find something we like more. One last try *vf:ProcessBlueprint*